### ISSUE TITLE
Job summaries in the quick-list are no longer proactively updated

### ISSUE THREAD

**Issue by ascobie** [Azure/BatchExplorer#1815]
Latest insider build

The job summaries in the quick-list are no longer being proactively updated. I have 2 jobs that are active and 1 that is terminating in the quick-list. I went out for a coffee and they were still in this state when I returned. It was not until I clicked on them that the states updated to completed. 

This will be rather annoying for customers that are running a lot of jobs as they will have to click into each one to see that it has completed successfully.

**Comment by timotheeguerin** [comment]
The list was never auto updating, you had to click the refresh at the top.

**Comment by ascobie** [comment]
I don't think this is the case. In the past i have had jobs run end-to-end and be completed in the quick-list without me having to click on them.

**Comment by timotheeguerin** [comment]
Not sure how this was happening, I pretty sure we never had the quick list updating automatically. The job you are currently browsing will update in the list but that's it.
